module/data/actor/character.mjs

```diff
@@ -125,6 +127,19 @@ export default class CharacterData extends CreatureTemplate {
     super._migrateData(source);
     AttributesFields._migrateInitiative(source.attributes);
   }
+
+  /* -------------------------------------------- */
+
+  prepareDerivedData() {
+    // Populate the background field with background if background item exists but is not set
+    if ( !(this.details.background instanceof dnd5e.documents.Item5e) ) {
+      const background = this.parent.itemTypes.background?.[0];
+      if ( background ) Object.defineProperty(this.details, "background", {
+        get() { return background; },
+        enumerable: false
+      });
```

Comment:
```suggestion
      if ( background ) Object.defineProperty(this.details, "background", {
        value: background,
        enumerable: false
      });
```

This would still miss the `toString` handler that we define on real `LocalDocumentField`s. I'm leaning towards just doing a system migration to convert this field rather than using `prepareDerivedData`. It should be quite simple and means we don't have to worry about these 'unmigrated' values lurking in any given Actor.

Reply:
Okay, I've swapped over to full migration.